app/views/devices_guidance/support_and_maintenance.md

```diff
@@ -102,4 +102,4 @@ We cannot increase data limits if a pupil has used 20GB within a month. Advise t
 
 ## How to contact DfE for help
 
-If you work for a local authority, academy trust, school or college and need help with any of the devices provided by this programme, [contact us](/get-support). We aim to respond within 5 working days.
+If you work for a local authority, academy trust, school or college and need help with any of the devices provided by this programme, [contact us](/get-support). We aim to respond within 3 working days.
```

Comment:
Do we need to mention the SLA here? It is mentioned on the get support home page. I think when the user is reading this all they need to know at this point is that they can contact us. I don't think they should be concerned about how long it's going to take us to respond. When they try to contact us, that's when they will want to know how long it may be before they can expect a response from us.

Reply:
Agreed